<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16090" started-at="2018-08-12T06:37:03Z" finished-at="2018-08-12T06:37:19Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16090" started-at="2018-08-12T06:37:03Z" finished-at="2018-08-12T06:37:19Z">
      <class name="PropertiesFileReader.MercuryDemo">
        <test-method status="PASS" signature="openbrowser()[pri:1, instance:PropertiesFileReader.MercuryDemo@5f282abb]" name="openbrowser" duration-ms="3373" started-at="2018-08-12T06:37:03Z" finished-at="2018-08-12T06:37:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method status="PASS" signature="openurl()[pri:2, instance:PropertiesFileReader.MercuryDemo@5f282abb]" name="openurl" duration-ms="4363" started-at="2018-08-12T06:37:07Z" finished-at="2018-08-12T06:37:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method status="PASS" signature="maxbrowser()[pri:3, instance:PropertiesFileReader.MercuryDemo@5f282abb]" name="maxbrowser" duration-ms="1136" started-at="2018-08-12T06:37:11Z" finished-at="2018-08-12T06:37:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- maxbrowser -->
        <test-method status="PASS" signature="loginvalid()[pri:4, instance:PropertiesFileReader.MercuryDemo@5f282abb]" name="loginvalid" duration-ms="5434" started-at="2018-08-12T06:37:12Z" description="verify that user able to enter mercurytours with valid data" finished-at="2018-08-12T06:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginvalid -->
        <test-method status="FAIL" signature="invalidlogin()[pri:5, instance:PropertiesFileReader.MercuryDemo@5f282abb]" name="invalidlogin" duration-ms="1695" started-at="2018-08-12T06:37:17Z" description="verify that user able to enter mercurytours with invalid data" finished-at="2018-08-12T06:37:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at PropertiesFileReader.MercuryDemo.invalidlogin(MercuryDemo.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidlogin -->
      </class> <!-- PropertiesFileReader.MercuryDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
